README.md

```diff
@@ -45,11 +51,13 @@ These can be overridden by setting the environment variables `RAILS_DEV_BOX_RAM`
 
 ## What's In The Box
 
+Ruby on Rails:
+
 * Development tools
 
 * Git
 
-* Ruby 2.7
+* Ruby 2.5
```

Comment:
Same here, should remain 2.7.

Reply:
Ubuntu 18.04 default ruby version is 2.5

Reply:
Ubuntu 18.04 default ruby version is 2.5